### PR TITLE
Add new file 

### PR DESCRIPTION
**Pull request by AnilChittem**
review the changes

### CODE REVIEW

File: open.txt
Comment (jaswanth424) [inline open.txt:1]: Please add some lines

**Review by jaswanth424 — CHANGES_REQUESTED** [review]

File: open.txt
Comment (AnilChittem) [reply open.txt:1]: hi added 2 new lines

**Review by AnilChittem — COMMENTED** [review]
request for changes

File: open.txt
Comment (jaswanth424) [reply open.txt:1]: Thanks for adding. Looks good.

**Review by jaswanth424 — APPROVED** [review]
You can merge to main branch.

**Pull request merged**